### ISSUE TITLE
`save()` broken for Lists

### ISSUE THREAD

**Issue by CGenie** [coleifer/walrus#30]
It seems that the fix from https://github.com/coleifer/walrus/issues/25 broke the `save()` method.

Here's what I do:

```python
class WithListModel(Model):
    name = TextField(index=True)
    values = ListField()

c = WithListModel.create(name='x')
c.values.append('x')
list(c.values)  # ['x']
c.save()
list(c.values)  # []
```

This is because of this part: https://github.com/coleifer/walrus/blob/master/walrus/models.py#L931 -- deleting now also deletes associated list...

**Comment by coleifer** [comment]
Oh shit, I think you're right.

**Comment by coleifer** [comment]
Should be fixed, 0.3.4 release pushed to PyPI with changes.
